config/ThingworxConnectorConfig.json

```diff
@@ -1,5 +1,5 @@
 {
    "LogLevel": 1,
-   "ThingworxBaseUrl": "https://setme.biz",
+   "ThingworxBaseUrl": "https://setme.biz/Thingworx/Things/ConnectorHost/Services/TakeInfo",
```

Comment:
Is this still considered the "base url" now that the endpoint is required?

Reply:
Good point - should be renamed